Базовая стилизация всех страниц

comeunne opened the pull request:
---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1755265/tasks/3)

---
:mortar_board: [Разметка внутренней страницы](https://up.htmlacademy.ru/htmlcss/33/user/1755265/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/1755265-sedona-33/2/

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1755265/tasks/5)

---
:mortar_board: [Базовая стилизация всех страниц](https://up.htmlacademy.ru/htmlcss/33/user/1755265/tasks/6)

comeunne committed:
Добавляет разметку для Главной страницы

comeunne committed:
Разметка Главной страницы и Каталога

Добавляет Логотип к Главной странице и разметку для страницы Каталога

comeunne committed:
Добавляет адрес к Главной странице у Каталога

Kanasimy commented on index.html:
```diff
@@ -3,10 +3,164 @@
+
+    <section class="advantages">
+    <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Канон!</p>
+     <li class="advantages-item">
```
Нет родительского тега ul

Kanasimy commented on index.html:
```diff
@@ -3,10 +3,164 @@
+    </ul>
+
+    <section class="advantages">
+       <li class="advantages-item">
```
так же нет ul

Kanasimy commented on catalog.html:
```diff
@@ -0,0 +1,276 @@
+  </li>
+  <li class="pagination-item">
+    <a class="pagination-link" href="#">10</a>
+    </a>
```
лишнее </a>

comeunne replied:
почему лишнее?

comeunne replied:
> лишнее
почему и что лишнее?

Kanasimy replied:
закрывающий тег `</a>`

Kanasimy reviewed (commented):
1) Ссылка с логотипа на текущую страницу не нужна, нужна ссылка в каталоге
2) Не во всех секциях есть заголовки
3) Неплохо бы оставлять более содержательное описание а не "Фоновый рисунок-1"
4) Телефон ссылкой с атрибутом tel
5) advantages - сделать одним списком со вложенными ul а не разбивать на отдельные секции
6) Section обычно применяется для больших блоков, помещать в секцию логотип немного странно

comeunne commented:
> 1. Ссылка с логотипа на текущую страницу не нужна, нужна ссылка в каталоге
> 2. Не во всех секциях есть заголовки –– уточните плиз, их у меня перебор или наоборот ?
> 3. Неплохо бы оставлять более содержательное описание а не "Фоновый рисунок-1"
> 4. Телефон ссылкой с атрибутом tel
> 5. advantages - сделать одним списком со вложенными ul а не разбивать на отдельные секции
> 6. Section обычно применяется для больших блоков, помещать в секцию логотип немного странно

comeunne committed:
Учитывает замечания и ошибки

Удалена ссылка у лого, удален и добавлен тег tel с adress к телефону, изменены названия изображений, секции переименованы в <ul>, логотип академии вне секции

comeunne committed:
Добавляет изображения

comeunne committed:
Добавляет изображения

Kanasimy commented on catalog.html:
```diff
@@ -0,0 +1,269 @@
+      <a class="footer-phone-number" href="tel:+78128121212">+7 (812) 812-12-12</a>
+    </adress>
+
+    <a href="https://htmlacademy.ru/"><img src="#" alt="Логотип академии"><img src="images/academy.svg" width="114" height="32"></a>
```
Два изображения логотипа

Kanasimy commented on index.html:
```diff
@@ -3,10 +3,151 @@
+    <h2><strong>Седона - небольшой городок в Аризоне, заслуживающий большего!</strong></h2>
+
+    <ul class="advantages">
+    <p>Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Канон!</p>
```
В ul может быть вложен только li, p нужно перенести в li

Kanasimy reviewed (commented):
1) Иконки - декоративные картинки, их будем стилями выводить или inline svg если нужна анимация
2) catalog-background.jpg - тоже стилями нужно добавлять 
3) У иконки бургера линии не голубые , видимо результат оптимизации
4) Посмотрите валидатор, он подскажет где ошибки, если не понятен текст ошибки - лучше писать в телеграмм быстрей отреагирую

comeunne committed:
Добавляет стили к Главной и Каталогу

Добавляет стили к Главной странице и к Каталогу

Kanasimy reviewed (commented):
1)По графике осталось -  Декоративные картинки остались тегами img - это иконки жилье, еда, сувениры, иконки в footer-social, иконки в навигации, фон catalog-background.jpg ...
Не у всех изображений есть атрибут alt.
2) Очень много ошибок в валидаторе, нужно поправить
3) Уберите везде абсолютное позиционирование - оно не нужно
4) Форматирование css должно соответствовать требованиям [кодгайда Академии](http://codeguide.academy/html-css.html).
5) Не нужно нумеровать классы aside-image-1, aside-image-2, достаточно aside-image

Задания нужно делать поэтапно, что бы потом не переделывать как с абсолютным позиционированием, в текущем задании позиционировать элементы не требовалось.

comeunne committed:
Добавляет стили и контент-изображения

Добавляет стили и крупные контент-изображения

comeunne committed:
Добавляет стили и контент-изображения

Kanasimy reviewed (commented):
Остается:
По декоративным изображением - все хорошо, только логотип картинкой
От абсолютного позиционирования остались размеры, и top, bottom и т.п. - они пока тоже не нужны, позиционировать будите с помощью флексом и гридов, и достаточно будет задать высоту родительскому блоки, и отступы дочерним
4) Форматирование css должно соответствовать требованиям [кодгайда Академии](http://codeguide.academy/html-css.html).
5) Не нужно нумеровать классы aside-image-1, aside-image-2, достаточно aside-image

Поправьте и отправляйте как дз, кнопка в личном кабинете академии

comeunne committed:
Убирает лишние стили у селекторов

comeunne committed:
Меняет наставника

Muhammadabumansur commented:
![image](https://user-images.githubusercontent.com/16342548/154338346-b4709a37-2a56-430f-af5a-b2d9d2a3b65a.png)

У этой картинки почему-то появилась обводка, можно ее вручную убрать из кода либо удали и экспортируй заново нормально

Muhammadabumansur commented:
![image](https://user-images.githubusercontent.com/16342548/154338472-7ecdd1a6-cf83-4dff-b2eb-e4606775a0c8.png)

Выделять папки исходя из страниц неправильно, можно выделять по смыслу схожие картинки

Muhammadabumansur commented:
А так, все ок

Muhammadabumansur commented:
Создай новый пулреквест и отправь задание **6. Базовая стилизация всех страниц** заново на проверку
